### DISCUSSION TITLE
UDP TS Module

### DISCUSSION THREAD

**Discussion by luk3skyw4lker** [denoland/deno#8714] [Q&A]
Hello! I am interested on building a Deno UDP module (just like Node has dgram), but I have no clue on how to start, would anyone give me a tip?

**Comment by lucacasonato** [comment]
Deno already has UDP support in unstable: https://doc.deno.land/builtin/unstable#Deno.listenDatagram

**Reply by luk3skyw4lker** [reply]
Nice, any plans on when it's going to be on stable?
